templates/vue/cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+    "env": {
+        "DEV_URL": "http://localhost:8888",
+        "TEST_TAPESTRY_NAME": "testing",
+        "USER_TAPESTRY_NAME": "testing-user",
+        "ADMIN_USERNAME": "admin",
+        "ADMIN_PASSWORD": "Tapestry123!",
+        "SUBSCRIBER_USERNAME": "narendras",
+        "SUBSCRIBER_PASSWORD": ")%AbNZgppS)uNvxT5)OdgzhX"
+    }
+}
```

Comment:
I think it's better we remove this file and add it to gitignore and just update the README to let the user know they need to add this file in this format. Otherwise, users might accidentally push their own settings to the repo, which we don't want.

Reply:
Sure, I'll add a `cypress-sample.json` and update the README